src/tests/ftest/soak/utils.py

```diff
@@ -316,7 +308,7 @@ def get_daos_server_logs(self):
             commands = ["scp {}:/var/tmp/daos_testing/daos*.log.* {}".format(host, daos_dir),
                         "scp {}:/var/tmp/daos_testing/daos*.log {}".format(host, daos_dir)]
             for command in commands:
-                run_command(command, timeout=120)
+                run_command(command, timeout=120, raise_exception=False)
```

Comment:
We should reinstate the error raising for non-tearDown calling and instead use a try .. except .. in the pre_tear_down() method to capture any errors and perform the appropriate action (log a message, or add the error to the list of errors to be reported at the end of tearDown, etc.)